proto/eth/v1/beacon_chain.proto

```diff
@@ -270,3 +270,12 @@ message DepositContract {
     // The address of the deployed deposit contract in use.
     string address = 2;
 }
+
+message WeakSubjectivityResponse {
+	WeakSubjectivityData data = 1;
+}
+
+message WeakSubjectivityData {
+        ethereum.eth.v1.Checkpoint ws_checkpoint = 1;
```

Comment:
formatting

Reply:
What formatting do you mean?

Reply:
it looks like this line is double indented